src/pymor/algorithms/rand_la.py

```diff
@@ -170,14 +170,24 @@ def rrf(A, source_product=None, range_product=None, q=2, l=8, return_rand=False,
 def random_generalized_svd(A, range_product=None, source_product=None, modes=6, p=20, q=2):
     r"""Randomized SVD of an |Operator|.
 
-    Viewing the `A` as a `A.dim x len(A)` matrix, the return value
-    of this method is the randomized singular value decomposition of `A`, where the
-    inner product on :math:`\mathbb{R}^{\mathtt{A.dim}}` is given by 'range_product' and
-    the inner product on :math:`\mathbb{R}^{\mathtt{len(A)}}` is given by `source_product`.
+    Viewing `A` as an :math:`m` by :math:`n` matrix, the return value
+    of this method is the randomized generalized singular value decomposition of `A`:
 
     .. math::
 
-        A = U \Sigma V^H \mathtt{source_product}
+        A = U \Sigma V^{-1},
+
+    where the inner product on the range :math:`\mathbb{R}^m` is given by
+
+    .. math::
+
+        \langle x, y \rangle_S = x^TSy
```

Comment:
```suggestion
        (x, y)_S = x^TSy
```

This is the notation, I have used in the 'Building a Reduced Basis' tutorial, and I'd say it's also more standard in mathematics. Pointy brackets are for dual pairings ...